src/libs/brands.ts

```diff
@@ -151,6 +155,10 @@ export const brands: Record<string, BrandConfiguration> = {
         light: '#e9ecef',
       },
     },
+    datasetBuilderSnapshotId:
+      getConfig().terraDeploymentEnv === 'dev'
```

Comment:
Should this be a switchcase? I think it has `dev`, `prod`, and `staging` as options.

Reply:
Right now we only have snapshots in `dev` and `prod`. It's true that this will build a bogus link in `staging` but I don't think that's an important case to handle here, as we don't use staging.